docs/amber_script.md

```diff
@@ -435,6 +438,27 @@ The following commands are all specified within the `PIPELINE` command.
   FRAMEBUFFER_SIZE _width_ _height_
 ```
 
+```groovy
+  # Require that subgroups must be launched with all invocations active
+  # for given shader. Only valid in the compute stage.
+  REQUIRE_FULL_SUBGROUPS {name_of_shader}
+```
+
+```groovy
+  # Allow SubgroupSize to vary for given shader.
+  VARYING_SUBGROUP_SIZE {name_of_shader}
+```
+
+```groovy
+  # Require a specific SubgroupSize the for given shader:
+  # |subgroup_size| can be set one of the values below:
+  #  - a power-of-two integer that _must_ be greater or equal to minSubgroupSize
+  #    and be less than or equal to maxSubgroupSize
+  # - "MIN" to set the required subgroup size to the minSubgroupSize
```

Comment:
Remove the `""`s around MIN and MAX

Reply:
I updated the syntax to match the form proposed in https://github.com/google/amber/pull/852#discussion_r418064490